test/gotrue_test.exs

```diff
@@ -154,7 +154,11 @@ defmodule GoTrueTest do
         %{
           method: :post,
           url: "http://auth.example.com/token?grant_type=password",
-          headers: [{"content-type", "application/json"}, authorization: "Bearer super-secret"],
+          headers: [
+            {"content-type", "application/json"},
+            apikey: "super-secret",
+            Authorization: "super-secret"
```

Comment:
both headers are using the same value, shouldn't they be different?

Reply:
No, only in the case where you actually have an access_token != the project api key, for example if you have a logged in user.
There are test cases where this happens: https://github.com/treebee/gotrue-elixir/blob/configurable-auth-header/test/gotrue_test.exs#L359

And initializing both, `apikey` and `authorization` with the projects apikey is what they also do in the JavaScript client: https://github.com/supabase/supabase-js/blob/master/src/SupabaseClient.ts#L147